src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,18 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE = 35;
+    private static final int MIN_PERIOD_IN_UKR = 10;
+
+    @Override
+    public boolean test(Candidate candidate) {
+        String[] parts = candidate.getPeriodsInUkr().split("-");
+        int periodsInUkr = (Integer.parseInt(parts[1]) - Integer.parseInt(parts[0]));
```

Comment:
```suggestion
        int periodsInUkr = Integer.parseInt(parts[1]) - Integer.parseInt(parts[0]);
```
Better make separate variables fromYear, toYear and make indexes constant